addons/main/functions/fnc_jammerPlayerLocal.sqf

```diff
@@ -108,7 +98,7 @@ if (!isNull _drone) then {
 		// less intense actions when drone is only approaching the jammer area (gives pilot time to react to the presence of the jammer)
 		if (isRemoteControlling player && (isNull curatorCamera)) then {	// if player uses UAV camera currently (and did not step into Zeus mode)
 			// calculate video image distortion
-			private _distDrone2killRadius = _distDroneToJammer - _radius;
+			private _distDrone2killRadius = _distDroneToJammer - (_radFalloff + _radEffective);
 			private _distDrone2pilot = _drone distance player;
 			private _sharpness = [0, 4, _distDrone2killRadius/_distDrone2pilot] call BIS_fnc_lerp;
```

Comment:
When you are within `_radFalloff` do a linear interpolation of `_sharpness` between `0` and `4`. 

Remove any reference to the pilot (just to clean up).